### PR TITLE
add abinit

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/abinit/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+./configure --prefix=${PREFIX}
+make
```
Comment (marcelotrevisani) [inline recipes/abinit/build.sh]: You can use multiple cores do compile this project if you want to
```suggestion
make ${CPU_COUNT}
```

**Comment by marcelotrevisani** [comment]
According to the documentation, it seems that there are some dependencies for this package
```
Abinit depends on several other packages:

  * BLAS (mandatory)
  * LAPACK (mandatory)
  * NetCDF (will become mandatory soon)
  * ETSF I/O (might become mandatory soon)
  * LibXC (recommended)
  * FoX (may be safely disabled)
  * AtomPAW (may safely be disabled)
  * BigDFT (recommended)
  * Wannier90 (recommended)
```

File: recipes/abinit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  sha256: b362bda0ccefbc7254d653d551ba1952aa4a92f9f845208caa3446cb8960ad76
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - make
```
Comment (marcelotrevisani) [inline recipes/abinit/meta.yaml]: You need ``autoconf`` as well
```suggestion
    - make
    - autoconf
```

File: recipes/abinit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
```
Comment (marcelotrevisani) [inline recipes/abinit/meta.yaml]: I think you will also need the ``c++`` compilers
```suggestion
    - {{ compiler('cxx') }}
    - {{ compiler('c') }}
    - make
```

**Comment by jan-janssen** [comment]
I have issues to download the package from the official source: 
```
ERROR: cannot verify www.abinit.org's certificate, issued by ‘/C=GB/ST=Greater Manchester/L=Salford/O=Sectigo Limited/CN=Sectigo RSA Domain Validation Secure Server CA’:
  Unable to locally verify the issuer's authority.
```

**Pull request closed without merging**